src/main/java/com/google/appengine/appcfg/AbstractAppCfgMojo.java

```diff
@@ -370,6 +377,10 @@ protected void executeAppCfgBackendsCommand(String action, String appDir)
       arguments.add("--enable_jar_classes");
     }
 
+    if (additionalParams != null && !additionalParams.isEmpty()) {
+      arguments.add(additionalParams);
```

Comment:
Not sure about the behavior when you want more than 1 flag and the space separator handled by the Java Process build list of arguments. I think to be 100% portable (Windows, Linux) we might have to do an arguments.add for each flag (black separated) of the additionalParams.